mythril/laser/ethereum/svm.py

```diff
@@ -149,6 +168,11 @@ def sym_exec(
                     "No contract was created during the execution of contract creation "
                     "Increase the resources for creation execution (--max-depth or --create-timeout)"
                 )
+            else:
+                # Replace previous initial state with new ones after contract creation
+                self.set_standard_initial_state(
```

Comment:
Why is this necessary. In this case the initial state would just be the one with the creator and attacker account right? The creation transaction is included in the transaction sequence

Reply:
Here the initial state would have the created account.

Reply:
That doesn't make much sense. Then the creation transaction will create another new account, if you try to replay the transaction sequence?

Reply:
The creation transaction isn't included in the steps I think.

Reply:
Ok, then this is okay